spec/builders/messages/facebook/message_builder_spec.rb

```diff
@@ -58,5 +58,88 @@
       expect(facebook_channel.inbox.reload.contacts.count).to eq(1)
       expect(contact.name).to eq(default_name)
     end
+
+    context 'when lock to single conversation' do
+      subject(:mocked_message_builder) { described_class.new(fb_message.new(contact_inbox.source_id, {}, '', '', ''), facebook_inbox).perform }
+
+      let!(:fb_message) { Struct.new(:sender_id, :attachments, :content, :identifier, :in_reply_to_external_id) }
```

Comment:
let's follow the same pattern which is used in the previous cases in this spec. 
Let's use factories instead of this struct. 

```
let!(:message_object) { build(:incoming_fb_text_message).to_json }
```

ref:  spec/factories/facebook_message/incoming_fb_text_message.rb